hpOneView/resources/activity/tasks.py

```diff
@@ -25,34 +25,18 @@
 standard_library.install_aliases()
 
 
-from hpOneView.resources.resource import ResourceClient
+from hpOneView.resources.resource import Resource, ResourcePatchMixin
 
 
-class Tasks(object):
+class Tasks(ResourcePatchMixin, Resource):
```

Comment:
ResorucePatchMixin not required here.
There is no point in inheriting from Resource class here as Task class doesn't support any other methods except Get.

Reply:
I am supporting get_by_id,patch endpoint also in tasks resource thats why I am inheriting Resource and ResourcePatchMixin classes.

Reply:
Got it. just checked the API doc of 1200 and there is one patch endpoint.